Clients/Xamarin.Interactive.Client.Mac/OutlineView/CollectionOutlineView.cs

```diff
@@ -121,8 +121,14 @@ public CollectionOutlineViewDataSource.NodeProxy GetNodeProxy (nint row)
             get { return GetNodeProxy (SelectedRow); }
             set {
                 nint row = -1;
+                var current = GetNodeProxy (SelectedRow);
```

Comment:
Current appears unused--how does this even build? Did you mean to use it below for `DeselectRow`?